ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md

```diff
@@ -63,7 +63,6 @@
     - [IP DHCP Relay](#ip-dhcp-relay)
     - [IP ICMP Redirect](#ip-icmp-redirect)
     - [LACP](#lacp)
-    - [Link Tracking Groups](#link-tracking-groups)
```

Comment:
Disappeared, seen in a different PR

Reply:
was an issue with yaml delimiter - fixed!